Can't make conda environment on Intel Mac (Tahoe 26.3.1a)

### Bug Description

When creating a new conda environment from `environment.yml` using `conda env create`, the solver hangs indefinitely (observed >47 minutes with no resolution).

```
conda env create -f environment.yml -n rmg_env18 -v
Channels:
 - conda-forge
 - rmg
Platform: osx-64
Collecting package metadata (repodata.json): done
Solving environment: |
```

The trigger for needing a fresh environment was the recent addition of `pysidt-rmg` to the dependencies.

### How To Reproduce

1. On macOS 26 (Tahoe) with an Intel (`osx-64`) machine, clone RMG-Py at or near commit `3.3.0-605-g17990e8ee`
2. Run `conda env create -f environment.yml -n rmg_env_new`
3. Observe that the solver never completes

### Expected Behavior

The conda environment should resolve and install within a few minutes, as it does on CI (macOS runners on GitHub Actions).

### Workaround

Rather than creating a fresh environment, I activated my previous environment (`rmg_env17`, Python 3.11.14) and installed the missing package directly:

```bash
conda install -c rmg pysidt-rmg
```

This succeeded, for me, and my environment is working.

But it leaves our [installation method](https://reactionmechanismgenerator.github.io/RMG-Py/users/rmg/installation/anacondaDeveloper.html) for others with a similar setup broken.

### Installation Information

- **OS**: macOS 26.3.1 alpha (`ProductVersionExtra: (a)`, Darwin 25.3.0) — Intel x86_64 (`osx-64` platform)
- **Installation method**: Source install from `environment.yml` via Miniconda3
- **conda version**: 26.1.1
  - solver: libmamba (default)
  - conda-libmamba-solver: 25.4.0
  - libmamba / libmambapy: 2.1.1
- **RMG-Py**: `3.3.0-605-g17990e8ee` (from `git describe --tags` on `main`)
- **RMG-database**: `3.3.0-22-g2c90c6985`

Key packages in the working `rmg_env17` environment (Python 3.11.14, `osx-64`):

| Package | Version | Channel |
|---|---|---|
| python | 3.11.14 | conda-forge |
| cython | 3.0.12 | conda-forge |
| numpy | 1.26.4 | conda-forge |
| scipy | 1.16.0 | conda-forge |
| rdkit | 2025.09.5 | conda-forge |
| openbabel | 3.1.1 | conda-forge |
| matplotlib | 3.10.8 | conda-forge |
| mopac | 23.1.2 | conda-forge |
| pydas | 1.0.4 | rmg |
| pydqed | 1.0.4 | rmg |
| pysidt-rmg | 1.2.0 | rmg |

### Additional Context

**Hypothesis (from Claude Code, not verified):** The hang is likely caused by the `__osx` virtual package being set to `26.3.1` (macOS Tahoe alpha). The libmamba solver may be spending a very long time trying to satisfy package constraints in conda-forge and the `rmg` channel that were authored before macOS 26 existed — for example, packages that specify an upper bound on `__osx` or that pin `osx_is_app_ext`. This would explain why CI on macOS (GitHub Actions runners on Sequoia/15.x) passed without issue. Other Intel Mac users on stable macOS releases may not hit this at all.

I note that updating conda itself didn't help (`conda update -n base -c defaults conda` upgraded from 26.1.0 to 26.1.1 but 26.3.2 appears to be the newest, which may require a newer Python in the base environment, which would mean reinstalling Miniconda entirely).

More bad news:

When I did `make` in the new environment it crashed with 
```running build_ext
building 'rmgpy.rmgobject' extension
clang -DNDEBUG -fwrapv -O2 -Wall -fPIC -O2 -isystem /Users/rwest/miniconda3/envs/rmg_env18/include -fPIC -O2 -isystem /Users/rwest/miniconda3/envs/rmg_env18/include -I/Users/rwest/miniconda3/envs/rmg_env18/lib/python3.11/site-packages/numpy/core/include -I/Users/rwest/miniconda3/envs/rmg_env18/include/python3.11 -c rmgpy/rmgobject.c -o build/temp.macosx-11.0-x86_64-cpython-311/rmgpy/rmgobject.o
clang: error: invalid version number in 'MACOSX_DEPLOYMENT_TARGET=11.0'
error: command '/usr/bin/clang' failed with exit code 1
```

I suspected polluted environment variables, so I started a new terminal, reactivated the new conda environment, and ran `make clean`.
but that now says `/Users/rwest/miniconda3/envs/rmg_env18/bin/python: No module named pip`.
Apparently `pip` is not in our `environment.yml`.  So I `conda install pip` and run `make clean` then `make` still crashes.

I tried `make build` to see if that's any easier than the full `make install`.  The output is simpler and shorter and I think points to the problem.
```
make build
python setup.py build_ext --inplace
Deduplicated LDFLAGS from '-Wl,-rpath,/Users/rwest/miniconda3/envs/rmg_env18/lib -L/Users/rwest/miniconda3/envs/rmg_env18/lib -Wl,-rpath,/Users/rwest/miniconda3/envs/rmg_env18/lib -L/Users/rwest/miniconda3/envs/rmg_env18/lib' to '-Wl,-rpath,/Users/rwest/miniconda3/envs/rmg_env18/lib -L/Users/rwest/miniconda3/envs/rmg_env18/lib'
Deduplicated LDSHARED from 'clang -bundle -undefined dynamic_lookup -Wl,-rpath,/Users/rwest/miniconda3/envs/rmg_env18/lib -L/Users/rwest/miniconda3/envs/rmg_env18/lib -Wl,-rpath,/Users/rwest/miniconda3/envs/rmg_env18/lib -L/Users/rwest/miniconda3/envs/rmg_env18/lib' to 'clang -bundle -undefined dynamic_lookup -Wl,-rpath,/Users/rwest/miniconda3/envs/rmg_env18/lib -L/Users/rwest/miniconda3/envs/rmg_env18/lib'
Deduplicated BLDSHARED from 'clang -bundle -undefined dynamic_lookup -Wl,-rpath,/Users/rwest/miniconda3/envs/rmg_env18/lib -L/Users/rwest/miniconda3/envs/rmg_env18/lib -Wl,-rpath,/Users/rwest/miniconda3/envs/rmg_env18/lib -L/Users/rwest/miniconda3/envs/rmg_env18/lib' to 'clang -bundle -undefined dynamic_lookup -Wl,-rpath,/Users/rwest/miniconda3/envs/rmg_env18/lib -L/Users/rwest/miniconda3/envs/rmg_env18/lib'
Deduplicated PY_LDFLAGS from '-Wl,-rpath,/Users/rwest/miniconda3/envs/rmg_env18/lib -L/Users/rwest/miniconda3/envs/rmg_env18/lib -Wl,-rpath,/Users/rwest/miniconda3/envs/rmg_env18/lib -L/Users/rwest/miniconda3/envs/rmg_env18/lib' to '-Wl,-rpath,/Users/rwest/miniconda3/envs/rmg_env18/lib -L/Users/rwest/miniconda3/envs/rmg_env18/lib'
Compiling rmgpy/rmgobject.pyx because it changed.
Compiling rmgpy/kinetics/arrhenius.pyx because it changed.
Compiling rmgpy/kinetics/chebyshev.pyx because it changed.
Compiling rmgpy/kinetics/kineticsdata.pyx because it changed.
Compiling rmgpy/kinetics/falloff.pyx because it changed.
Compiling rmgpy/kinetics/model.pyx because it changed.
Compiling rmgpy/kinetics/tunneling.pyx because it changed.
Compiling rmgpy/kinetics/surface.pyx because it changed.
Compiling rmgpy/kinetics/uncertainties.pyx because it changed.
Compiling rmgpy/molecule/atomtype.py because it changed.
Compiling rmgpy/molecule/element.py because it changed.
Compiling rmgpy/molecule/graph.pyx because it changed.
Compiling rmgpy/molecule/group.py because it changed.
Compiling rmgpy/molecule/molecule.py because it changed.
Compiling rmgpy/molecule/symmetry.py because it changed.
Compiling rmgpy/molecule/vf2.pyx because it changed.
Compiling rmgpy/molecule/converter.py because it changed.
Compiling rmgpy/molecule/translator.py because it changed.
Compiling rmgpy/molecule/util.py because it changed.
Compiling rmgpy/molecule/inchi.py because it changed.
Compiling rmgpy/molecule/resonance.py because it changed.
Compiling rmgpy/molecule/pathfinder.py because it changed.
Compiling rmgpy/molecule/kekulize.pyx because it changed.
Compiling rmgpy/pdep/collision.pyx because it changed.
Compiling rmgpy/pdep/configuration.pyx because it changed.
Compiling rmgpy/pdep/me.pyx because it changed.
Compiling rmgpy/pdep/msc.pyx because it changed.
Compiling rmgpy/pdep/reaction.pyx because it changed.
Compiling rmgpy/pdep/rs.pyx because it changed.
Compiling rmgpy/pdep/cse.pyx because it changed.
Compiling rmgpy/statmech/conformer.pyx because it changed.
Compiling rmgpy/statmech/mode.pyx because it changed.
Compiling rmgpy/statmech/rotation.pyx because it changed.
Compiling rmgpy/statmech/schrodinger.pyx because it changed.
Compiling rmgpy/statmech/torsion.pyx because it changed.
Compiling rmgpy/statmech/translation.pyx because it changed.
Compiling rmgpy/statmech/vibration.pyx because it changed.
Compiling rmgpy/thermo/thermodata.pyx because it changed.
Compiling rmgpy/thermo/model.pyx because it changed.
Compiling rmgpy/thermo/nasa.pyx because it changed.
Compiling rmgpy/thermo/wilhoit.pyx because it changed.
Compiling rmgpy/constants.py because it changed.
Compiling rmgpy/quantity.py because it changed.
Compiling rmgpy/reaction.py because it changed.
Compiling rmgpy/species.py because it changed.
Compiling rmgpy/chemkin.pyx because it changed.
Compiling rmgpy/solver/base.pyx because it changed.
Compiling rmgpy/solver/simple.pyx because it changed.
Compiling rmgpy/solver/liquid.pyx because it changed.
Compiling rmgpy/solver/mbSampled.pyx because it changed.
Compiling rmgpy/solver/surface.pyx because it changed.
[ 1/51] Cythonizing rmgpy/chemkin.pyx
[ 2/51] Cythonizing rmgpy/constants.py
[ 3/51] Cythonizing rmgpy/kinetics/arrhenius.pyx
[ 4/51] Cythonizing rmgpy/kinetics/chebyshev.pyx
[ 5/51] Cythonizing rmgpy/kinetics/falloff.pyx
[ 6/51] Cythonizing rmgpy/kinetics/kineticsdata.pyx
[ 7/51] Cythonizing rmgpy/kinetics/model.pyx
[ 8/51] Cythonizing rmgpy/kinetics/surface.pyx
[ 9/51] Cythonizing rmgpy/kinetics/tunneling.pyx
[10/51] Cythonizing rmgpy/kinetics/uncertainties.pyx
[11/51] Cythonizing rmgpy/molecule/atomtype.py
[12/51] Cythonizing rmgpy/molecule/converter.py
[13/51] Cythonizing rmgpy/molecule/element.py
[14/51] Cythonizing rmgpy/molecule/graph.pyx
[15/51] Cythonizing rmgpy/molecule/group.py
[16/51] Cythonizing rmgpy/molecule/inchi.py
[17/51] Cythonizing rmgpy/molecule/kekulize.pyx
[18/51] Cythonizing rmgpy/molecule/molecule.py
[19/51] Cythonizing rmgpy/molecule/pathfinder.py
[20/51] Cythonizing rmgpy/molecule/resonance.py
[21/51] Cythonizing rmgpy/molecule/symmetry.py
[22/51] Cythonizing rmgpy/molecule/translator.py
[23/51] Cythonizing rmgpy/molecule/util.py
[24/51] Cythonizing rmgpy/molecule/vf2.pyx
[25/51] Cythonizing rmgpy/pdep/collision.pyx
[26/51] Cythonizing rmgpy/pdep/configuration.pyx
[27/51] Cythonizing rmgpy/pdep/cse.pyx
[28/51] Cythonizing rmgpy/pdep/me.pyx
[29/51] Cythonizing rmgpy/pdep/msc.pyx
[30/51] Cythonizing rmgpy/pdep/reaction.pyx
[31/51] Cythonizing rmgpy/pdep/rs.pyx
[32/51] Cythonizing rmgpy/quantity.py
[33/51] Cythonizing rmgpy/reaction.py
[34/51] Cythonizing rmgpy/rmgobject.pyx
[35/51] Cythonizing rmgpy/solver/base.pyx
[36/51] Cythonizing rmgpy/solver/liquid.pyx
[37/51] Cythonizing rmgpy/solver/mbSampled.pyx
[38/51] Cythonizing rmgpy/solver/simple.pyx
[39/51] Cythonizing rmgpy/solver/surface.pyx
warning: rmgpy/solver/surface.pyx:408:61: Unreachable code
[40/51] Cythonizing rmgpy/species.py
[41/51] Cythonizing rmgpy/statmech/conformer.pyx
[42/51] Cythonizing rmgpy/statmech/mode.pyx
[43/51] Cythonizing rmgpy/statmech/rotation.pyx
[44/51] Cythonizing rmgpy/statmech/schrodinger.pyx
[45/51] Cythonizing rmgpy/statmech/torsion.pyx
[46/51] Cythonizing rmgpy/statmech/translation.pyx
[47/51] Cythonizing rmgpy/statmech/vibration.pyx
[48/51] Cythonizing rmgpy/thermo/model.pyx
[49/51] Cythonizing rmgpy/thermo/nasa.pyx
[50/51] Cythonizing rmgpy/thermo/thermodata.pyx
[51/51] Cythonizing rmgpy/thermo/wilhoit.pyx
/Users/rwest/miniconda3/envs/rmg_env18/lib/python3.11/site-packages/setuptools/dist.py:334: InformationOnly: Normalizing '4.0.0.rc1' to '4.0.0rc1'
  self.metadata.version = self._normalize_version(self.metadata.version)
running build_ext
building 'rmgpy.rmgobject' extension
creating build/temp.macosx-11.0-x86_64-cpython-311/rmgpy
clang -DNDEBUG -fwrapv -O2 -Wall -fPIC -O2 -isystem /Users/rwest/miniconda3/envs/rmg_env18/include -fPIC -O2 -isystem /Users/rwest/miniconda3/envs/rmg_env18/include -I/Users/rwest/miniconda3/envs/rmg_env18/lib/python3.11/site-packages/numpy/core/include -I/Users/rwest/miniconda3/envs/rmg_env18/include/python3.11 -c rmgpy/rmgobject.c -o build/temp.macosx-11.0-x86_64-cpython-311/rmgpy/rmgobject.o
clang: error: invalid version number in 'MACOSX_DEPLOYMENT_TARGET=11.0'
error: command '/usr/bin/clang' failed with exit code 1
make: *** [build] Error 1
```

--- a/.github/workflows/CI.yml
+++ b/.github/workflows/CI.yml
@@ -80,6 +80,14 @@ jobs:
       - name: Checkout RMG-Py
         uses: actions/checkout@v6
 
+      - name: Pin Python version for CI matrix
+        # environment.yml pins python=3.11 for fast user installs; rewrite to the
+        # matrix version here so setup-miniconda can inject the correct pin cleanly.
+        # The -i.bak / rm pattern works on both GNU sed (Linux) and BSD sed (macOS).
+        run: |
+          sed -i.bak "s/conda-forge::python =3\.11/conda-forge::python =${{ matrix.python-version }}/" environment.yml
+          rm -f environment.yml.bak
+
       - name: Setup Miniforge Python ${{ matrix.python-version }}
         uses: conda-incubator/setup-miniconda@v4
         with:
@@ -147,13 +155,13 @@ jobs:
       - name: Checkout RMG-Py
         uses: actions/checkout@v6
 
-      - name: Setup Miniforge Python 3.9
+      - name: Setup Miniforge Python 3.11
         uses: conda-incubator/setup-miniconda@v4
         with:
           environment-file: environment.yml
           miniforge-variant: Miniforge3
           miniforge-version: latest
-          python-version: 3.9
+          python-version: 3.11
           activate-environment: rmg_env
           auto-update-conda: true
           show-channel-urls: true

--- a/documentation/source/users/rmg/installation/anacondaDeveloper.rst
+++ b/documentation/source/users/rmg/installation/anacondaDeveloper.rst
@@ -4,6 +4,35 @@
 Installation by Source Using Anaconda Environment for Unix-based Systems: Linux and Mac OSX
 *******************************************************************************************
 
+.. warning::
+
+   **Before you begin — check your build tools are up to date.**
+
+   RMG-Py compiles Cython extensions during installation, so your C compiler and
+   conda installation must be reasonably modern or the build will fail.
+
+   **On macOS:** Ensure your Xcode Command Line Tools are current.  An outdated
+   installation (more than a year or two old) will cause ``make`` to fail with a
+   ``clang: error: invalid version number`` message.  To check::
+
+       clang --version
+
+   If the reported version is older than Apple clang 13, reinstall the tools::
+
+       sudo rm -rf /Library/Developer/CommandLineTools
+       xcode-select --install
+
+   **conda version:** Ensure conda is version 24.0 or newer (run ``conda --version``
+   to check).  Older versions use a slower solver that can hang indefinitely when
+   creating the RMG environment.  To update::
+
+       conda update -n base -c conda-forge conda
+
+   If ``conda update`` does not advance the version, install a specific release
+   explicitly::
+
+       conda install -n base -c conda-forge conda=26.3
+
 #. Install the `conda` package manager via `miniforge`, if you do not already have it (or Anaconda), by following the `Miniforge installation instructions <https://github.com/conda-forge/miniforge?tab=readme-ov-file#install>`_.
 
 #. If your `conda` version is older than 23.10.0, manually switch the solver backend to `libmamba` (or update your conda)::
@@ -40,6 +69,7 @@ Installation by Source Using Anaconda Environment for Unix-based Systems: Linux 
       These are a set of packages relevant for software development which have been bundled together by Apple.
       The easiest way to install this is to simply run one of the commands in the terminal, e.g. ``git``.
       The terminal will then prompt you to install the Command Line Tools.
+      See the warning at the top of this page about keeping the tools up to date.
 
 #. Install the latest versions of RMG and RMG-database through cloning the source code via Git. Make sure to start in an
    appropriate local directory where you want both RMG-Py and RMG-database folders to exist.
@@ -51,7 +81,7 @@ Installation by Source Using Anaconda Environment for Unix-based Systems: Linux 
 
    It is still possible to clone the repositories using ``https`` if you are
    unfamiliar with ``ssh``::
-   
+
     git clone https://github.com/ReactionMechanismGenerator/RMG-Py.git
     git clone https://github.com/ReactionMechanismGenerator/RMG-database.git
 
@@ -61,7 +91,10 @@ Installation by Source Using Anaconda Environment for Unix-based Systems: Linux 
 
     cd RMG-Py
 
-#. Create the conda environment for RMG-Py::
+#. Create the conda environment for RMG-Py.
+
+   The ``environment.yml`` file pins Python 3.11, so the solver only needs to
+   consider packages for a single Python version and the solve completes quickly::
 
     conda env create -f environment.yml
 
@@ -89,10 +122,10 @@ Installation by Source Using Anaconda Environment for Unix-based Systems: Linux 
        - nodefaults
      channel_priority: strict
      custom_channels:
-       main: null
-       r: null
-       anaconda: null
-       msys2: null
+         main: null
+         r: null
+         anaconda: null
+         msys2: null
 
 #. Activate conda environment ::
 

--- a/documentation/source/users/rmg/installation/anacondaDeveloper.rst
+++ b/documentation/source/users/rmg/installation/anacondaDeveloper.rst
@@ -4,41 +4,27 @@
 Installation by Source Using Anaconda Environment for Unix-based Systems: Linux and Mac OSX
 *******************************************************************************************
 
-.. warning::
-
-   **Before you begin — check your build tools are up to date.**
+#. Before you begin — check your build tools are up to date.
 
    RMG-Py compiles Cython extensions during installation, so your C compiler and
    conda installation must be reasonably modern or the build will fail.
 
-   **On macOS:** Ensure your Xcode Command Line Tools are current.  An outdated
-   installation (more than a year or two old) will cause ``make`` to fail with a
-   ``clang: error: invalid version number`` message.  To check::
-
-       clang --version
-
-   If the reported version is older than Apple clang 13, reinstall the tools::
+   **On MacOS:** Ensure your Xcode Command Line Tools are current.  If more than a year or two old, reinstall the tools::
 
        sudo rm -rf /Library/Developer/CommandLineTools
        xcode-select --install
 
-   **conda version:** Ensure conda is version 24.0 or newer (run ``conda --version``
-   to check).  Older versions use a slower solver that can hang indefinitely when
-   creating the RMG environment.  To update::
+   **conda version:** Ensure conda is also up to date.  To update::
 
        conda update -n base -c conda-forge conda
 
-   If ``conda update`` does not advance the version, install a specific release
-   explicitly::
+   If ``conda update`` does not advance the version (which can happen), install a specific release explicitly, e.g.::
 
        conda install -n base -c conda-forge conda=26.3
 
-#. Install the `conda` package manager via `miniforge`, if you do not already have it (or Anaconda), by following the `Miniforge installation instructions <https://github.com/conda-forge/miniforge?tab=readme-ov-file#install>`_.
+#. If you do not already have the `conda` package manager (or Anaconda) then install it via `miniforge`, by following the `Miniforge installation instructions <https://github.com/conda-forge/miniforge?tab=readme-ov-file#install>`_.
 
-#. If your `conda` version is older than 23.10.0, manually switch the solver backend to `libmamba` (or update your conda)::
 
-    conda install -n base conda-libmamba-solver
-    conda config --set solver libmamba
 
 #. There are a few system-level dependencies which are required and should not be installed via Conda. These include
    `Git <https://git-scm.com/>`_ for version control, `GNU Make <https://www.gnu.org/software/make/>`_, and the C and C++ compilers from the `GNU Compiler Collection (GCC) <https://gcc.gnu.org/>`_ for compiling RMG.
@@ -69,7 +55,7 @@ Installation by Source Using Anaconda Environment for Unix-based Systems: Linux 
       These are a set of packages relevant for software development which have been bundled together by Apple.
       The easiest way to install this is to simply run one of the commands in the terminal, e.g. ``git``.
       The terminal will then prompt you to install the Command Line Tools.
-      See the warning at the top of this page about keeping the tools up to date.
+      See the warning in step 1 about keeping the tools up to date.
 
 #. Install the latest versions of RMG and RMG-database through cloning the source code via Git. Make sure to start in an
    appropriate local directory where you want both RMG-Py and RMG-database folders to exist.
@@ -93,8 +79,7 @@ Installation by Source Using Anaconda Environment for Unix-based Systems: Linux 
 
 #. Create the conda environment for RMG-Py.
 
-   The ``environment.yml`` file pins Python 3.11, so the solver only needs to
-   consider packages for a single Python version and the solve completes quickly::
+   To create an environment called ``rmg_env`` containing all that you need for RMG, run::
 
     conda env create -f environment.yml
 
@@ -102,17 +87,13 @@ Installation by Source Using Anaconda Environment for Unix-based Systems: Linux 
 
     conda env create -f environment.yml -n rmg_env2
 
-   If either of these commands return an error due to being unable to find the ``conda`` command,
-   try to either close and reopen your terminal to refresh your environment variables
-   or type the following command.
+   If you have a recent version of conda (``conda -V`` reports >= 26.3) then the preferred command is without the ``env`` subcommand::
 
-   If on Linux or pre-Catalina MacOS (or if you have a bash shell)::
+    conda create --file environment.yml --name rmg_env
 
-    source ~/.bashrc
-
-   If on MacOS Catalina or later (or if you have a Z shell)::
-
-    source ~/.zshrc
+   If any of these commands return an error due to being unable to find the ``conda`` command,
+   (for example if you only just installed it)
+   then close and reopen your terminal (or log out and log in again) to refresh your environment variables.
 
    NOTE: You may wish to forbid ``conda`` from installing from the Anaconda channels due to licensing restrictions.
    The ``environment.yml`` file already forbids using default channels, but you may further add the file ``.condarc`` to your RMG-Py directory (or modify the ``.condarc`` in your home directory) with the following contents before running the ``conda env create`` command: ::
@@ -152,7 +133,7 @@ Installation by Source Using Anaconda Environment for Unix-based Systems: Linux 
 
     Installing Julia and ReactionMechanismSimulator.jl (RMS) will enable all the features in RMG that require RMS-based reactors,
     as well as using ``method='ode'`` when solving the Master Equation with Arkane.
-    Note that installing RMS can cause errors when running Cantera simulations; this should not affect normal RMG use, but if you wish to run Cantera simulations you will need to maintain a separate conda environment without RMS in it.
+    Note that installing RMS has sometimes caused errors when running Cantera; this should not affect normal RMG use, but if you wish to run Cantera simulations you might need to use a separate conda environment without RMS in it.
 
     Ensure that you have modified your environment variables as described above, and then run the following: ::
 

--- a/documentation/source/users/rmg/installation/anacondaDeveloper.rst
+++ b/documentation/source/users/rmg/installation/anacondaDeveloper.rst
@@ -9,9 +9,8 @@ Installation by Source Using Anaconda Environment for Unix-based Systems: Linux 
    RMG-Py compiles Cython extensions during installation, so your C compiler and
    conda installation must be reasonably modern or the build will fail.
 
-   **On MacOS:** Ensure your Xcode Command Line Tools are current.  If more than a year or two old, reinstall the tools::
+   **On MacOS:** Ensure your Xcode Command Line Tools are current. If they are more than a year or two old, either download the latest Command Line Tools from `Apple Developer Downloads <https://developer.apple.com/download/all/>`_ (free Apple ID required) or manually remove them and reinstall them using::
 
-       sudo rm -rf /Library/Developer/CommandLineTools
        xcode-select --install
 
    **conda version:** Ensure conda is also up to date.  To update::

--- a/environment.yml
+++ b/environment.yml
@@ -28,12 +28,9 @@ dependencies:
   - conda-forge::xlrd
   - conda-forge::xlwt
   - conda-forge::h5py
-  # keep graphviz from conda-forge; see https://github.com/ReactionMechanismGenerator/RMG-Py/issues/2750
   - conda-forge::graphviz >=12
   - conda-forge::markupsafe
   - conda-forge::psutil
-  # conda-forge not default, since default has a version information bug
-  # (see https://github.com/ReactionMechanismGenerator/RMG-Py/pull/2421)
   - conda-forge::ncurses
   - conda-forge::suitesparse
   # ThermoCentralDatabaseInterface fails if pyopenssl is too old. 20 is just a guess at the version number. 
@@ -60,6 +57,7 @@ dependencies:
   - conda-forge::pydot
   - conda-forge::jinja2
   - conda-forge::jupyter
+  - conda-forge::pip
   - conda-forge::pymongo
   - conda-forge::pyparsing
   - conda-forge::pyyaml

--- a/environment.yml
+++ b/environment.yml
@@ -50,7 +50,7 @@ dependencies:
   - rmg::pysidt-rmg >=1.2
 
 # Python tools
-  - conda-forge::python >=3.9,<3.12  # leave as GEQ so that GitHub actions can add EQ w/o breaking (contradictory deps)
+  - conda-forge::python =3.11  # pinned to a single version to keep solver fast; CI rewrites this to the matrix version (see CI.yml)
   - conda-forge::setuptools <80
   - conda-forge::coverage
   - conda-forge::cython >=0.25.2,<3.1

